### DISCUSSION TITLE
Using Swiper in Next.js: ReferenceError: document is not defined

### DISCUSSION THREAD

**Discussion by powerseed** [nolimits4web/swiper#7476] [Q&A]
![image](https://github.com/nolimits4web/swiper/assets/42278237/4e3193ad-35eb-4b3b-9356-6e8b937e32dd)

I following the tutorial https://swiperjs.com/get-started, but saw this error.

Could someone help? Thanks!

Here is a similar question: https://stackoverflow.com/questions/72855330/referenceerror-swiper-is-not-defined-white-using-next-js

**Comment by naasif08** [comment]
Was your problem solve? I'm having same issue.
